### ISSUE TITLE
MVP Proposal Comments

### ISSUE THREAD

**Issue by loschorts** [jnvbui/SurveyPanda#1]
Hey Joseph,

Nice proposal! Here are a few comments: 

- You will want to budget more time for Phase 1 (Auth). This is tricky; I suggest 2 days.

- You'll likely need several more models to complete a survey: `questions`, `answers`, `responses`.

-	`answers` don't need to belong to `surveys` if they belong to `questions` and `questions` belong to `surveys`.

- Can you clarify what a `form` is?

**Comment by jnvbui** [comment]
Hey Ken, 

- I just added another day to phase 1. 

-  I already assigned questions, answers and responses as models in my schema. Do you mean I need more models other than those 3?

- I removed survey_id from answers since answers belongs to questions which belongs to survey.

- A form is like the actual survey that users can send out to participants to submit. When a participant submits actual survey form, the user's dashboard and survey summary gets updated to show a new response.

**Comment by jnvbui** [comment]
Also, I was wondering how I should show the responses. I was thinking 2 tabs where one tab has the individual responses sorted by date that someone submits a form and the other tab has all the responses with answers.

**Comment by loschorts** [comment]
Cool thanks for clarifying. I think your README.md didn't have them originally, but I did see them in the Schema.  

I like the two tab approach: 1st tab is an index of responses, 2nd tab has meta-data: answer distribution for each question, etc. BTW, are you going to use all multiple choice?

**Comment by jnvbui** [comment]
No, I was going to let the user choose which type of question they wanna create. I was planning on allowing: multiple choice, checkbox, short answer(input type="text"), long answer(textarea), and dropdown. 

For the 2nd tab, I was only going to do something like this:

For fixed answer choices like multiple choice, checkbox, dropdown:
Question: Favorite movie?
Answers: option1 (23), option2 (15), option3 (17), etc. where the values 23,15, 17 are total responses for that answer.

I don't think i'll have enough time to put these on charts/graphs but that will be part of my bonus feature.

For open-ended answers like short answer and long answer:
Question: What makes you the perfect fit?
Response1: "blah blah blah"
Response2: "blah blah blah"

I feel like this will make it easier to follow for the user analyzing the results.

**Comment by jnvbui** [comment]
I was also thinking maybe I could make the answers for the open-ended ones all hidden and the user can click show to view the results. This way the user won't have to scroll a lot if they have many responses.

**Comment by loschorts** [comment]
OK that sounds good. I like the hidden answers idea too.

**Comment by jnvbui** [comment]
Added Question, Answer and Response model task to Phase Checklist (Phase 2)
